packages/clerk-js/src/ui/elements/ArrowBlockButton.tsx

```diff
@@ -91,25 +93,32 @@ export const ArrowBlockButton = (props: ArrowBlockButtonProps) => {
           )}
         </Flex>
       )}
-      <Text
-        elementDescriptor={textElementDescriptor}
-        elementId={textElementId}
-        as='span'
-        truncate
-        colorScheme='inherit'
-        variant='buttonSmallRegular'
-        localizationKey={textLocalizationKey}
+      <Flex
```

Comment:
Hey George, this PR looks great. I'd like to check whether this extra div here introduces any layout shifting when the button moves in and out of the loading state, then we can merge

Reply:
Looks like adding a new div messes with the flex layout of the button, please see the screenshot below: 
![image](https://user-images.githubusercontent.com/1811063/193233469-4b3ece23-cbe4-44b1-bf88-029b361feeda.png)

Reply:
Should be fixed. Take a look.